mat64/matrix.go

```diff
@@ -63,16 +63,28 @@ type VectorSetter interface {
 	SetCol(i int, src []float64) int
 }
 
-// A RowViewer can return a slice of float64 reflecting a row that is backed by the matrix
-// data.
+// A RowViewer can return a Vector reflecting a row that is backed by the matrix
+// data. The Vector returned should have len(nCols).
```

Comment:
"Vector returned should have len(nCols)." This doesn't make sense; it is not possible to call len on a Vector. Lower case Vector and s/should/will/?